.github/workflows/test-spec.yml

```diff
@@ -28,45 +28,47 @@ jobs:
             admin_password: admin
           - version: 2.0.0
             admin_password: admin
-          - version: 2.19.1
-          - version: 2.19.1
+          - version: 3.0.0-beta1
+          - version: 3.0.0-beta1
```

Comment:
Do not change these as we need to test the latest released version most of the time.